pkgs/applications/emulators/dosbox-staging/default.nix

```diff
@@ -110,10 +110,13 @@ stdenv.mkDerivation rec {
       existing DOSBox codebase while leveraging modern development tools and
       practices.
     '';
-    license = licenses.gpl2Plus;
-    maintainers = with maintainers; [ joshuafern AndersonTorres ];
-    platforms = platforms.unix;
+    changelog = "https://github.com/dosbox-staging/dosbox-staging/releases/tag/v${self.version}";
+    license = lib.licenses.gpl2Plus;
+    maintainers = builtins.attrValues {
```

Comment:
why are we doing this?

Reply:
Getting rid of `with`.

Reply:
I think there's nothing wrong with usage of `with` at such a local scope, and I don't think there is a good alternative until something like [RFC0110](https://github.com/NixOS/rfcs/pull/110) is implemented. Using `attrValues` here makes it a lot less readable to me.